packages/nodes-base/nodes/Pushover/Pushover.node.ts

```diff
@@ -250,12 +250,15 @@ export class Pushover implements INodeType {
 						description: "Your message's title, otherwise your app's name is used",
 					},
 					{
-						displayName: 'Timestamp',
-						name: 'timestamp',
-						type: 'dateTime',
-						default: '',
+						displayName: 'Time to Live (Seconds)',
+						name: 'ttl',
+						type: 'number',
+						typeOptions: {
+							minValue: 0,
+						},
+						default: 0,
```

Comment:
I'm not sure if `0` works as a good default as it isn't documented but I couldn't find an easy way from the other properties to just exclude it if it isn't set.

Reply:
Based on https://github.com/Emka877/pushover-rs/commit/08a768e104fe8d4694199dea7fabf6ea8b975973 it sounds like we need to exclude passing the field if it's not set as opposed to using a default.